src/App/Service/Github.php

```diff
@@ -273,7 +273,19 @@ public function isPRValid(array $pullRequest, Filters $filters): bool
             return false;
         }
 
-        // Filter Repository PRovaite
+        // Filter Repository Name
+        if ($filters->hasFilter(Filters::FILTER_REPOSITORY_NAME)) {
+            if ($filters->isFilterIncluded(Filters::FILTER_REPOSITORY_NAME)
+                && !in_array($pullRequest['repository']['name'], $filters->getFilterData(Filters::FILTER_REPOSITORY_PRIVATE), true)) {
+                return false;
+            }
+            if (!$filters->isFilterIncluded(Filters::FILTER_REPOSITORY_NAME)
+                && in_array($pullRequest['repository']['name'], $filters->getFilterData(Filters::FILTER_REPOSITORY_PRIVATE), true)) {
```

Comment:
```suggestion
                && in_array($pullRequest['repository']['name'], $filters->getFilterData(Filters::FILTER_REPOSITORY_NAME), true)) {
```

But I don't get the double check.
Only the condition is needed imho

Reply:
Fixed